modules/eco-setting-node-maintenance-cr-cli.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-maintenance-cr-cli_{context}"]
+= Setting a node to maintenance mode by using the CLI
+
+You can put a node into maintenance mode with a `NodeMaintenance` custom resource (CR). When you apply a `NodeMaintenance` CR, all allowed pods are evicted and the node is rendered unschedulable. Evicted pods are queued to be moved to another node in the cluster.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in to the cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the following node maintenance CR, and save the file as `nodemaintenance-cr.yaml`:
++
+[source,yaml]
+----
+apiVersion: nodemaintenance.medik8s.io/v1beta1
+kind: NodeMaintenance
+metadata:
+  name: maintenance-example  <1>
+spec:
+  nodeName: node-1.example.com <2>
+  reason: "NIC replacement" <3>
+----
+<1> The name of the node maintenance CR
+<2> The name of the node to be put into maintenance mode
+<3> A plain text description of the reason for maintenance
++
+.  Apply the node maintenance CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f nodemaintenance-cr.yaml
```

Comment:
Should this filename be the same as the filename you used in the step 1 example (`maintenance-example`)?

Reply:
I think it is already the same or I a missing something.
In step 1 we create a CR file, `nodemaintenance-cr.yaml`, where the CR's name is `maintenance-example` and in step 2 we apply/create the CR by it's file name, `nodemaintenance-cr.yaml`.

Reply:
Updated the metadata.name for consistency.